--- a/src/qt.mk	Mon Feb 21 21:18:11 2011 +0100
+++ b/src/qt.mk	Tue Feb 22 00:17:48 2011 +0100
@@ -10,7 +10,7 @@
 $(PKG)_FILE     := $(PKG)-everywhere-opensource-src-$($(PKG)_VERSION).tar.gz
 $(PKG)_WEBSITE  := http://qt.nokia.com/
 $(PKG)_URL      := http://get.qt.nokia.com/qt/source/$($(PKG)_FILE)
-$(PKG)_DEPS     := gcc libodbc++ postgresql freetds openssl libgcrypt zlib libpng jpeg libmng tiff sqlite libiconv dbus
+$(PKG)_DEPS     := gcc libodbc++ freetds libgcrypt zlib libpng jpeg libmng tiff sqlite libiconv dbus
 
 define $(PKG)_UPDATE
     wget -q -O- 'http://qt.gitorious.org/qt/qt/commits' | \
@@ -26,8 +26,6 @@
     #
     # For shared Qt with qt-zlib, add -lQtCore4 to end of OPENSSL_LIBS to satisfy zlib dependency.
     cd '$(1)' && \
-        OPENSSL_LIBS="`'$(TARGET)-pkg-config' --libs-only-l openssl`" \
-        PSQL_LIBS="-lpq -lsecur32 `'$(TARGET)-pkg-config' --libs-only-l openssl` -lws2_32" \
         SYBASE_LIBS="-lsybdb -liconv -lws2_32" \
         ./configure \
         -opensource \
@@ -47,6 +45,7 @@
         -no-gstreamer \
         -no-phonon \
         -no-phonon-backend \
+        -no-openssl \
         -accessibility \
         -no-reduce-exports \
         -no-rpath \
@@ -56,7 +55,6 @@
         -nomake examples \
         -qt-sql-sqlite \
         -qt-sql-odbc \
-        -qt-sql-psql \
         -qt-sql-tds -D Q_USE_SYBASE \
         -system-zlib \
         -system-libpng \
@@ -64,7 +62,6 @@
         -system-libtiff \
         -system-libmng \
         -system-sqlite \
-        -openssl-linked \
         -dbus-linked \
         -v
 
