--- a/src/qt.mk Thu Dec 29 19:46:44 2011 +0100
+++ b/src/qt.mk Fri Dec 30 23:44:38 2011 +0100
@@ -10,7 +10,7 @@
 $(PKG)_FILE     := $(PKG)-everywhere-opensource-src-$($(PKG)_VERSION).tar.gz
 $(PKG)_WEBSITE  := http://qt.nokia.com/
 $(PKG)_URL      := http://get.qt.nokia.com/qt/source/$($(PKG)_FILE)
-$(PKG)_DEPS     := gcc libodbc++ postgresql freetds openssl zlib libpng jpeg libmng tiff sqlite dbus
+$(PKG)_DEPS     := gcc libodbc++ freetds zlib libpng jpeg libmng tiff sqlite dbus

 define $(PKG)_UPDATE
     wget -q -O- 'http://qt.gitorious.org/qt/qt/commits' | \
@@ -23,8 +23,6 @@
     cd '$(1)' && QTDIR='$(1)' ./bin/syncqt

     cd '$(1)' && \
-        OPENSSL_LIBS="`'$(TARGET)-pkg-config' --libs-only-l openssl`" \
-        PSQL_LIBS="-lpq -lsecur32 `'$(TARGET)-pkg-config' --libs-only-l openssl` -lws2_32" \
         SYBASE_LIBS="-lsybdb `'$(TARGET)-pkg-config' --libs-only-l gnutls` -liconv -lws2_32" \
         ./configure \
         -opensource \
@@ -45,6 +43,7 @@
         -no-gstreamer \
         -no-phonon \
         -no-phonon-backend \
+        -no-openssl \
         -accessibility \
         -no-reduce-exports \
         -no-rpath \
@@ -54,7 +53,6 @@
         -nomake examples \
         -qt-sql-sqlite \
         -qt-sql-odbc \
-        -qt-sql-psql \
         -qt-sql-tds -D Q_USE_SYBASE \
         -system-zlib \
         -system-libpng \
@@ -62,7 +60,6 @@
         -system-libtiff \
         -system-libmng \
         -system-sqlite \
-        -openssl-linked \
         -dbus-linked \
         -v
